src/extension/views/json/json.js

```diff
@@ -220,6 +220,32 @@ export class JSONView extends LitElement {
     return elements;
   }
 
+  /**
+   * Finds the row with the most keys and returns key names as headers.
+   * @param {Object[]} rows The rows
+   * @returns {string[]} The header names
+   */
+  getHeaders(rows) {
+    const sortedRows = rows.sort((a, b) => Object.keys(b).length - Object.keys(a).length);
+    return Object.keys(sortedRows[0]);
+  }
+
+  /**
+   * Sorts all rows based on the headers
+   * @param {Object[]} rows The rows
+   * @param {string[]} headers The header names
+   * @returns {Object[]} The sorted rows
+   */
+  sortRows(rows, headers) {
```

Comment:
misleading name. better: `matchColsToHeaders()`  ??

Reply:
Actually, `sortColumns` would be more accurate :)